Main.java

```diff
@@ -1,3 +1,4 @@
+import java.util.Scanner;
 /**
  *
  * @author 

```

Comment:
Don't forget to add the program description and your name inside of the code header!